src/handlers/web3.js

```diff
@@ -22,6 +26,20 @@ export const web3Instance = new Web3(
   new Web3.providers.HttpProvider(`https://mainnet.infura.io/`),
 );
 
+export let ens;
+if (typeof window.web3 !== 'undefined') {
+  console.log('web3 browser detected, using.');
+  window.web3.version.getNetwork(function(err, network) {
+    if (err) {
+      console.log(err);
+    }
+    ens = new ENS({
+      provider: web3Instance.currentProvider,
+      network: network,
+    });
```

Comment:
Because you're not using window web3 there is no requirement to perform this check. This can simply be as follows:
```
export const ens = new ENS({
      provider: web3Instance.currentProvider,
      network: network,
    });
```

However when setting a new Web3 Provider, this ens instance should reflect that as well, see next comment

Reply:
On the next function `webSetHttpProvider` it needs to reassign ens with the new currentProvider, which can simply be provided using the provider parameter

```
ens = new ENS({
      provider: provider,
      network: network,
    });
```

Reply:
Hey @pedrouid thanks for the review, where do we get the network from? Since previously we were fetching it from -
`window.web3.version.getNetwork(function(err, network) { .. }`

Reply:
Okay you mean to remove this check? `if (typeof window.web3 !== 'undefined') {.. }`